firefox-ios/Client/Nimbus/NimbusFeatureFlagLayer.swift

```diff
@@ -341,10 +338,6 @@ final class NimbusFeatureFlagLayer {
         return nimbus.features.pdfRefactorFeature.value().enabled
     }
 
-    private func checkPreferSwitchToOpenTabOverDuplicate(from nimbus: FxNimbus) -> Bool {
-        return nimbus.features.homescreenFeature.value().preferSwitchToOpenTab
-    }
```

Comment:
Thank you for the PR, we also need to remove the `"prefer-switch-to-open-tab"` feature flag under the `.yaml` file [here](https://github.com/mozilla-mobile/firefox-ios/blob/39ab164a9e4cd315f3740bb6c69ac36de1f06f7f/firefox-ios/nimbus-features/homescreenFeature.yaml#L15-L36).